ivy_tests/test_ivy/test_functional/test_experimental/test_core/test_manipulation.py

```diff
@@ -1015,3 +1015,46 @@ def test_expand(
         x=x[0],
         shape=shape,
     )
+
+
+@st.composite
+def _concat_from_sequence_helper(draw):
+    shape = draw(helpers.get_shape(min_num_dims=1))
+    num_arrays = draw(st.integers(min_value=1, max_value=3))
+    arrays, _ = helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("valid"),
+        shape=shape,
+        num_arrays=num_arrays,
+        ret_shape=True,
+    )
+    axis = draw(helpers.get_axis(shape))
+    return arrays, axis
+
+
+# concat_from_sequence
+@handle_test(
+    fn_tree="functional.ivy.experimental.concat_from_sequence",
+    arrays_axis=_concat_from_sequence_helper(),
+    new_axis=st.integers(min_value=0, max_value=1),
+)
+def test_concat_from_sequence(
+    *,
+    arrays_axis,
+    new_axis,
+    test_flags,
+    backend_fw,
+    fn_name,
+    on_device,
+    ground_truth_backend,
+):
+    arrays, axis = arrays_axis
+    helpers.test_function(
+        ground_truth_backend=ground_truth_backend,
+        test_flags=test_flags,
```

Comment:
You shouldn't have removed the `input_dtypes` argument. You need to get a `dtype_arrays_axis` in `handle_test`. You can get the dtype in the composite strategy this way `dtypes, arrays = draw(helpers.dtype_and_values...`.